v4/source/jobs.yaml

```diff
@@ -0,0 +1,171 @@
+  '/jobs':
+    get:
+      tags:
+        - jobs
+      summary: Get the jobs.
+      description: |
+        Get a page of jobs. Use the query parameters to modify the behaviour of this endpoint.
+        __Minimum server version: 4.1__
+        ##### Permissions
+        Must have `manage_system` permission.
```

Comment:
0/5 on this, but wondering if there's any point adding new ones at this stage, given what's wanted might change when speccing advanced permissions.